commands/mta_command.go

```diff
@@ -68,14 +73,14 @@ func (c *MtaCommand) Execute(args []string) ExecutionStatus {
 		terminal.EntityNameColor(context.Space), terminal.EntityNameColor(context.Username))
 
 	// Create new REST client
-	mtaClient, err := c.NewMtaClient(host)
+	mtaV2Client, err := c.NewMtaV2Client(host)
```

Comment:
I do not remember but have we targeted some backwards compatibility?

Reply:
the v2 backend is in itself backwards compatible (or should be, at least)